Lib9c/Action/RuneSummon.cs

```diff
@@ -0,0 +1,223 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Data;
+using System.Linq;
+using Bencodex.Types;
+using Lib9c;
+using Lib9c.Abstractions;
+using Libplanet.Action;
+using Libplanet.Action.State;
+using Libplanet.Crypto;
+using Libplanet.Types.Assets;
+using Nekoyume.Action.Exceptions;
+using Nekoyume.Extensions;
+using Nekoyume.Model.State;
+using Nekoyume.TableData;
+using Nekoyume.TableData.Summon;
+using Serilog;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Action
+{
+    [ActionType("rune_summon")]
+    public class RuneSummon : GameAction, IRuneSummonV1
+    {
+        public const string AvatarAddressKey = "aa";
+        public Address AvatarAddress;
+
+        public const string GroupIdKey = "gid";
+        public int GroupId;
+
+        public const string SummonCountKey = "sc";
+        public int SummonCount;
+
+        private const int SummonLimit = 10;
+        public const int RuneQuantity = 100;
```

Comment:
IMHO, it seems quite inconvenient to change quantity of rune summoned.
Can we separate it to Sheet data or somewhere easier to manage?